README.adoc

```diff
@@ -83,6 +81,36 @@ Will produce this output:
 
 This macro also supports `.lutaml` files.
 
+Instead of using the direct path to the file one can use `lutaml-express-index` document attribute to supply directory with express files or yaml index file to parse as well as the cache file location. The syntax is as follows:
+
+[source,adoc]
+-----
+:lutaml-express-index: my_custom_name; dir_or_index_path[; cache=cache_path]
+-----
+
+Where:
+
+* `my_custom_name` - is name of the parsed express files context for the later use with lutaml macro
+
+* `dir_or_index_path` - location of directory with express files or path to yaml index file to parse
+
+* `cache_path` - optional, location of the expressir cache file to use
+
+Example of real usage:
+
+[source,adoc]
+-----
+= Document title
+Author
+:lutaml-express-index: my_custom_name; /path/to/express_files; cache=/path/to/cache_file.yaml
+
+[lutaml,my_custom_name,my_context]
+----
+== {{schema.id}}
```

Comment:
Currently we pass schema list as context. Eg if exp file contains 5 schemas then `lutaml` block will be rendered 5 times each time with different schema object as context.

>And how can I access multiple schemas within one lutaml block?

Currently you can't do that, each block has access to only one schema. This changed because of the new expressir `from_files` schema parsing feature. Need to implement this feature.

Reply:
Returned the old foramt: context == exp repository